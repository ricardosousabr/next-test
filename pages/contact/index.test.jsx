import { render } from '@testing-library/react'
import Contact from './index'

it('renders homepage unchanged', () => {
  const { container } = render(<Contact />)
  expect(container).toMatchSnapshot()
})