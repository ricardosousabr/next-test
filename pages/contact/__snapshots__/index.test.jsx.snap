// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a
          href=""
        >
          Linkedin
        </a>
      </li>
      <li>
        <a
          href=""
        >
          Intagram
        </a>
      </li>
      <li>
        <a
          href=""
        >
          Facebook
        </a>
      </li>
    </ul>
  </div>
</div>
`;
